fdbserver/workloads/DataLossRecovery.actor.cpp

```diff
@@ -134,7 +134,7 @@ struct DataLossRecoveryWorkload : TestWorkload {
 				} else {
 					tr.clear(key);
 				}
-				wait(timeoutError(tr.commit(), 30.0));
```

Comment:
So it could take longer than that to commit the transaction?

Reply:
Yes. In the failed seed, I saw TLog failed to make progress, and was killed after 20s. Then there is a long recovery. This timeout was thrown and failed the test. With the fix, the transaction is retried and succeeds.